src/main/java/com/commercetools/sync/types/utils/TypeUpdateLocalizedEnumActionUtils.java

```diff
@@ -0,0 +1,91 @@
+package com.commercetools.sync.types.utils;
+
+import com.commercetools.sync.commons.exceptions.DuplicateKeyException;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedEnumValue;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.commands.updateactions.AddLocalizedEnumValue;
+import io.sphere.sdk.types.commands.updateactions.ChangeLocalizedEnumValueOrder;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildAddEnumValuesUpdateActions;
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildChangeEnumValuesOrderUpdateAction;
+import static java.util.Collections.emptyList;
+
+public final class TypeUpdateLocalizedEnumActionUtils {
```

Comment:
I would name it `LocalizedEnumUpdateActionUtils`

Reply:
https://github.com/commercetools/commercetools-sync-java/commit/6f1484541de42487b69083d4556201bc2cb40080#diff-b2b8cf70cc2c073798ff3fe0c0df17f8L21